src/main/user-impl/java/com/tidb/jdbc/TidbCdcOperate.java

```diff
@@ -30,81 +65,158 @@ public static TidbCdcOperate of(ConnectionImpl connection,Ticdc ticdc){
         return new TidbCdcOperate(connection,ticdc);
     }
 
-    public TidbCdcOperate refreshSnapshot(){
-        String useTicdcACID = getTidbSnapshotParameter(TIDB_USE_TICDC_ACID_KEY,null);
+
+
+    private Boolean isRun(){
+        String useTicdcACID = this.ticdc.getUseTicdcACID();
         if(useTicdcACID == null){
-            return this;
+            return false;
         }
         if(!"true".equals(useTicdcACID)){
+            return false;
+        }
+        return true;
+    }
```

Comment:
String useTicdcACID = this.ticdc.getUseTicdcACID();
return "true".equals(useTicdcACID);